config.go

```diff
@@ -200,6 +205,7 @@ type bitcoindConfig struct {
 	RPCPass        string `long:"rpcpass" default-mask:"-" description:"Password for RPC connections"`
 	ZMQPubRawBlock string `long:"zmqpubrawblock" description:"The address listening for ZMQ connections to deliver raw block notifications"`
 	ZMQPubRawTx    string `long:"zmqpubrawtx" description:"The address listening for ZMQ connections to deliver raw transaction notifications"`
+	EstimateMode   string `long:"estimatemode" description:"The fee estimate mode. Must be either ECONOMICAL or CONSERVATIVE(default)."`
```

Comment:
We don't need to specify the default here since the config library handles that for us.